packages/amplication-prisma-db/prisma/migrations/20230531171626_/migration.sql

```diff
@@ -0,0 +1,3 @@
+-- AlterTable
+ALTER TABLE "Project" ADD COLUMN     "demoRepoName" TEXT,
+ADD COLUMN     "useDemoRepo" BOOLEAN NOT NULL DEFAULT false;
```

Comment:
How many projects do we have? This will lock up the projects table to update all existing records with "false" one by one